### PR TITLE
Versioning Guidelines

### PR DESCRIPTION
**Pull request by damienjburks**
#### Summary

This PR contains a new document that contains the guidelines for how we go about versioning our artifacts. This was derived from our discussion: #268

Let me know if you have any questions, comments, or concerns.

### CODE REVIEW

File: docs/governance/community-guidelines/versioning.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+This document is a [community guideline].
+
+## Purpose
+
+This document outlines the versioning standards and practices to be followed within the Common Cloud Controls (CCC) project.
+
+## Ownership
+
+The [Delivery WG] is entrusted with the overall responsibility for managing version control across all artifacts within this project. This includes defining versioning standards, ensuring consistent application of those standards across various deliverables, and maintaining the integrity of version history. The [Delivery WG] collaborates closely with other [WG]s to ensure that all artifacts are versioned appropriately and in alignment with the project's release and update schedules that is set by the [Communications WG].
```
Comment (sshiells-scottlogic) [inline docs/governance/community-guidelines/versioning.md:11]: We need to add links to [Delivery WG] and [Communications WG]  at the link definitions at the bottom?

**Review by sshiells-scottlogic — COMMENTED** [review]
Looks good to me. Just some missing links then I am happy for this to be merged.

File: docs/governance/community-guidelines/versioning.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Artifact-Specific Versioning
+
+Versioning will be scoped to each artifact delivered by the working groups. This means each artifact may have its own unique version number, reflecting its specific timeline and updates.
+
+## Release Frequency
+
+Releases should happen, at most, one time per month. This schedule ensures a manageable release cadence and maintains the stability of our artifacts. For more information about the releases, please refer to this document: [Releases](./releases.md)
+
+[WG]: ../community-groups.md#working-groups
+[Communications WG]: ../working-groups/communications/charter.md
```
Comment (eddie-knight) [inline docs/governance/community-guidelines/versioning.md:26]:
```suggestion
[Communications WG]: ../working-groups/communications/charter.md
[Delivery WG]: ../working-groups/delivery/charter.md
```

File: docs/governance/community-guidelines/versioning.md (hunk 1 above)
Comment (eddie-knight) [reply docs/governance/community-guidelines/versioning.md:11]: Added!

**Comment by damienjburks** [comment]
@sshiells-scottlogic  updated it 😄

File: docs/governance/community-guidelines/versioning.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+The [Delivery WG] is entrusted with the overall responsibility for managing version control across all artifacts within this project. This includes defining versioning standards, ensuring consistent application of those standards across various deliverables, and maintaining the integrity of version history. The [Delivery WG] collaborates closely with other [WG]s to ensure that all artifacts are versioned appropriately and in alignment with the project's release and update schedules that is set by the [Communications WG].
+
+## CalVer Standard
+
+The CalVer Standard (Calendar Versioning) is a method of version control that utilizes the calendar date to label releases. This approach provides clear, time-based context to the version numbers, making it easier to understand when a particular version was released. For example, a version like `Object Storage 24.07` indicates that the release occurred in July 2024.
+
+## Artifact-Specific Versioning
+
+Versioning will be scoped to each artifact delivered by the working groups. This means each artifact may have its own unique version number, reflecting its specific timeline and updates.
```
Comment (eddie-knight) [inline docs/governance/community-guidelines/versioning.md:19]: We reviewed past discussion on today's WG call... is there a way to capture the fact that each release will contain all relevant assets? (features, threats, controls, test descriptions)

Comment (damienjburks) [reply docs/governance/community-guidelines/versioning.md]: Just updated it - would you like to take a peek to see if it's something you were looking for?

**Review by eddie-knight — APPROVED** [review]

**Pull request merged**